rust/processor/src/models/token_v2_models/v2_token_activities.rs

```diff
@@ -243,66 +245,26 @@ impl TokenActivityV2 {
                     is_fungible_v2: Some(false),
                     transaction_timestamp: txn_timestamp,
                 }));
-            } else {
```

Comment:
Oops I think that we might still need this for backfilling?

Reply:
yes let me add the `else` back in